tests/generate_openapi.py

```diff
@@ -8,17 +8,17 @@
 
 def generate_openapi(filename: str | Path = "openapi.json"):
     with open(filename, "w") as f:
-        json.dump(
-            get_openapi(
-                title=app.title,
-                version=app.version,
-                openapi_version=app.openapi_version,
-                description=app.description,
-                routes=app.routes,
-            ),
-            f,
-            indent=2,
+        spec_dict = get_openapi(
+            title=app.title,
+            version=app.version,
+            openapi_version=app.openapi_version,
+            description=app.description,
+            routes=app.routes,
         )
+        # Filter proxy functions
+        del spec_dict["paths"]["/{path}"]
```

Comment:
did you try the include_in_schema=False on the endpoint? https://fastapi.tiangolo.com/advanced/path-operation-advanced-configuration/#exclude-from-openapi